lib/Runtime/Math/WasmMath.inl

```diff
@@ -105,7 +105,21 @@ inline int WasmMath::Eqz(T value)
 template<>
 inline double WasmMath::Copysign(double aLeft, double aRight)
 {
+#if _M_IX86
+    double a = *(double*)Js::NumberConstants::AbsDoubleCst;
+    double b = *(double*)Js::NumberConstants::SgnDoubleBitCst;
+    __asm {
+        movq xmm0, aLeft;
```

Comment:
in fact, i think we should be able to do this in c++ and not use inline asm at all...

---
In reply to: [132787155](https://github.com/Microsoft/ChakraCore/pull/3514#discussion_r132787155) [](ancestors = 132787155)